packages/shared/src/lib/kratos.ts

```diff
@@ -209,6 +209,17 @@ export const getKratosFlow = async (
   return res.json();
 };
 
+export const getKratosSettingsFlow = async (
+  flow: AuthFlow,
+  id: string,
+): Promise<InitializationData> => {
+  const res = await fetch(`${authUrl}/self-service${flow}/flows?id=${id}`, {
+    credentials: 'include',
+    headers: { Accept: 'application/json' },
+  });
+  return res.json();
+};
+
```

Comment:
I think this is the same as the generic `getKratosFlow`

Reply:
Nope the format is different, really annoying.

Reply:
Oh, wow. The only difference is the search parameter. Lol.